### PR TITLE
Puppeteer/Playwright: Common use cases section + two new lessons

### PR DESCRIPTION
**Pull request by mstephen19**

### CODE REVIEW

**Comment by mstephen19** [comment]
Preview [here](https://developers.apify.com/academy/puppeteer-playwright/common-use-cases/logging-into-a-website?version=puppeteer-playwright)

**Comment by mstephen19** [comment]
Ready for review

File: content/academy/puppeteer_playwright/common_use_cases/logging_into_a_website.md [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+        await sendEmailContext.close();
+    })()
+);
+
+await Promise.all(promises);
+
+await browser.close();
+</marked-tab>
+```
+
```
Comment (metalwarrior665) [inline content/academy/puppeteer_playwright/common_use_cases/logging_into_a_website.md:393]: 1. Not sure what you agreed with this lesson but I would also add Crawlee code. That is what 90% of our users will be using so they need to be able to almost copy/paste that.
2. Mention that we can also pass in cookies to the actor and only inject them
(if you think these do not belong here, we should add them elsewhere)

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]

File: content/academy/puppeteer_playwright/common_use_cases/logging_into_a_website.md (hunk 1 above)
Comment (mstephen19) [reply content/academy/puppeteer_playwright/common_use_cases/logging_into_a_website.md:393]: Screenshotting your comment + adding it to backlog.

**Pull request merged**